index.js

```diff
@@ -2487,6 +2487,20 @@
   }
   S.rights = def('rights', {}, [$.Array($Either(a, b)), $.Array(b)], rights);
 
+  //# tagBy :: (a -> Boolean) -> a -> Either a a
+  //.
+  //. Takes a predicate and a value and returns a Right of the value if it
+  //. satisfies the predicate and a Left of the value otherwise.
+  //.
+  //. ```javascript
+  //. > S.compose(S.bimap(S.dec, S.inc), S.tagBy(odd))(5)
+  //. Right(6)
+  //. ```
```

Comment:
The motivating example is nice, but let's also include trivial examples:

```javascript
> S.tagBy(S.odd, 0)
Left(0)

> S.tagBy(S.odd, 1)
Right(1)
```

Also, please replace `odd` with `S.odd` in the current example.